addons/web/models/models.py

```diff
@@ -476,8 +476,8 @@ def search_panel_select_range(self, field_name, **kwargs):
         field = self._fields[field_name]
         supported_types = ['many2one', 'selection']
```

Comment:
Are these types surfaced anywhere in the interface? We may want to translate them as well to match e.g. how do they appear in Studio in non-english locales?

Reply:
```py
types = dict(self.env["ir.model.fields"]._fields["ttype"]._description_selection(self.env))
raise UserError(_(
    'Only types %(supported_types)s are supported for category (found type %(field_type)s)',
    supported_types=", ".join(types[t] for t in supported_types),
    field_type=types[field.type],
))
```
not the easiest but should work